docs/gettingstarted/index.md

```diff
@@ -17,6 +17,11 @@ From Zero to Modding
 5. Choose your IDE: Forge explicitly supports developing with Eclipse or IntelliJ environments, but any environment, from Netbeans to vi/emacs, can be made to work.
     * For Eclipse, you should run `gradlew eclipse` - this will download some more artifacts for building eclipse projects and then place the eclipse project artifacts in your current directory.
     * For IntelliJ, simply import the build.gradle file.
+
+!!! note    
+
+    `setupDecompWorkspace` is deleted (tested in `1.12.2 - 14.23.5.2860`) and you can skip step 4.
+
```

Comment:
This is not how you write docs. You can just copy the section text from one of the other branches which uses FG3 (1.13-1.15). We do not need to special case for people using FG2 here and keep the old setup stuff.

Reply:
I copied the section from 1.13.x branch, does it look like ok?